assets/scss/components/_service-info.scss

```diff
@@ -195,3 +195,48 @@ Styleguide Service Info.Logged In
 .last-login__more-details {
   margin: em(5) 0 0 0;
 }
+
+/*
+Floated items
+
+Service Info with `.service-info__floated-items` allows for content to be floated on the left and right sides with adequate margins. On mobile breakpoints content is displayed block
+
+Markup:
+<div class="service-info ">
+  <div class="service-info__floated-items">
+    <div class="service-info__left-item">
+      <span>Left Side</span>
+    </div>
+    <div class="service-info__right-item">
+      <span>Right Side</span>
+    </div>
+  </div>
+</div>
+
+Styleguide Service Info.Left and Right Items
+*/
+.service-info__floated-items {
```

Comment:
The 2 selectors that handle the float left and right at the moment are .logo and .service-info__help respectively. These 2 are coupled with widths of 33% and 66%. The idea of the .service-info__left-item and .service-info__right-item was just to float content left and right allowing the content to be as wide as needs be (especially he right hand side).

Agree entirely with adding the padding to .service-info instead of .service-info__floated-items I did this here to sandbox the padding so as not to affect anyone else's use of .service-info. 

If and when the refactor of service info happens I would expect the padding to be taken out of .service-info__floated-items and added to .service-info